<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="search_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>链接/关键词</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="page_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>页数</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="page_lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="filename_lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="filename_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>文件名(仅链接)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="link_pushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>链接</string>
   </property>
  </widget>
  <widget class="QPushButton" name="google_pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>谷歌</string>
   </property>
  </widget>
  <widget class="QPushButton" name="baidu_pushButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>百度</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zip_pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>压缩</string>
   </property>
  </widget>
  <widget class="QPushButton" name="count_pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>文件总合</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>211</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
